public/static/pathways/upenn_her2_early_stage_pathway.json

```diff
@@ -0,0 +1,589 @@
+{
+    "name": "HER2+ Early Stage Pathway",
+    "description": "Pathway for the Early Stage HER2+ Breast Cancer Pathway",
+    "library": "mCODE_Library.cql",
+    "version": 1.0,
+    "criteria": [
+      {
+        "elementName": "Condition",
+        "expected": "Breast Cancer",
+        "cql": "\"Primary Cancer Condition\" PrimaryCancer return Tuple{ value: PrimaryCancer.code.text.value, match: PrimaryCancer.code.text.value ~ 'Malignant neoplasm of breast (disorder)' } "
```

Comment:
Let me preface this by saying I'm a CQL newbie.

I noticed that the quoted part of `"\"Primary Cancer Condition\" PrimaryCancer ` is not surrounded by `[]`.  Should it be?

e.g. on line 41 I see `"[Observation: \"Primary tumor.clinical [Class] Cancer code\"] TLarge`.

Reply:
Nope this is not needed. So the `Primary Cancer Condition` is defined within the library. This is the same CQL as the other her2+ pathway so you can verify it works there too.